src/security/expect-ct.conf

```diff
@@ -0,0 +1,29 @@
+# ----------------------------------------------------------------------
+# | Expect-CT                                                          |
+# ----------------------------------------------------------------------
+
+# Opt-in to Certificate Transparency (CT).
+#
+# Certificate Transparency helps guard against several types of certificate-
+# based threats, including misissued certificates, maliciously acquired
+# certificates, and rogue CAs (Certificate Authorities).
+#
+# (!) When a site requires Certificate Transparency (using the `enforce`
+#     directive), they are requesting that the browser check that any
+#     certificate for that site appears in public CT logs (1).
+#
+# It is advised (2) to only serve the Expect-CT header via a secure connection
+# which can be done with either `env=https` or `"expr=%{HTTPS} == 'on'"`.
+# The exact way depends on your environment and might just be tried.
+#
+# (1) https://www.certificate-transparency.org/known-logs
+# (2) https://httpwg.org/http-extensions/expect-ct.html#http-request-type
+#
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT
+# https://www.certificate-transparency.org/resources-for-site-owners
+# https://groups.google.com/forum/#!forum/certificate-transparency
+# https://httpwg.org/http-extensions/expect-ct.html
+
+<IfModule mod_headers.c>
+    Header always set Expect-CT "max-age=86400; enforce" "expr=%{HTTPS} == 'on'"
```

Comment:
This line only allows the header to be set over a secure connection, this may need to change depending on resolution in https://github.com/h5bp/server-configs-apache/issues/194.

Reply:
Resolved by https://github.com/h5bp/server-configs-apache/issues/194#issuecomment-508881332.